content/posts/building-a-project-with-ractor-1.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Building a project with Ractor (part 1)
+date: 2023-08-02
+language: en
+author: Andy Pfister
+tags:
+  - ruby
+  - ractor
+---
+
+Hi!
```

Comment:
I would propose to re-work a bit the introduction. 
1. Usually you don't see a greeting in the blog posts - at least I never saw it at Simplificator posts :)
2. Since it's your first contribution to the blog, it's a bit illogical to say "long time no see" - technically, readers have never seen you (or read from you) 😅